src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -115,6 +121,45 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     }
   }
 
+  /* Return value here is a tuple: (workflow name, call name)
+   * TODO: This should use a more sophisticated way of resolving FQNs.  See DSDEEPB-986
+   */
+  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Try[(String, String)] = {
+    callFqn.split("\\.").toSeq match {
+      case s: Seq[String] if s.size >= 2 => Success(s.head, s.last)
+      case _ => Failure(new UnsupportedOperationException("Expected a fully qualified name to have at least two parts"))
+    }
+  }
+
+  private def callStdoutStderr(workflowId: WorkflowId, callFqn: String): Future[StdoutStderr] = {
+    for {
+      _ <- assertWorkflowExistence(workflowId)
+      _ <- assertCallExistence(workflowId, callFqn)
+      (wf, call) <- Future.fromTry(assertCallFqnWellFormed(callFqn))
```

Comment:
Oh nm, I did this on line 149 but that explicitly materializes the exception from `Try` with a `get` and then wraps that back in a `Try`.